include/modules/dg600f.h

```diff
@@ -38,7 +38,7 @@
 	#define COIN_ACCEPTOR_BAUDRATE 9600
 #endif
 
-namespace coinAcceptor {
+namespace dg600f {
```

Comment:
Header guard in this file should be changed:
```cpp
#ifndef BLESKOMAT_MODULES_DG600F_H
#define BLESKOMAT_MODULES_DG600F_H
```

Reply:
Good catch! I fixed it.